plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -0,0 +1,690 @@
+
+        self.keymap = self.keymaping(self.keymap, ap_config_data[0])
+
+        ap_selected_field = self.payload["config"]["display_fields"]["ap_selected_field"]
+        if ap_selected_field == "" or ap_selected_field == "all":
+            return ap_config_data
+        return  self.data_frame(ap_selected_field, ap_config_data)
+
+    def compare_ap_cofig_with_inputdata(self, apconfig):
+        """
+        This function used to compare with the input ap detail with the current ap configuration
+        information are not same, those data will be updated in the AP input information.
+        Parameters:
+          - apconfig: This is response of the get_ap_configuration
+        Returns:
+            This will be the return the final data for update AP detail.
+            "final_input": [
+                {
+                    "adminStatus": true,
+                    "apList": [
+                        {
+                            "macAddress": "34:5d:a8:0e:20:b4"
+                        }
+                    ],
+                    "configureLedBrightnessLevel": true,
+                    "ledBrightnessLevel": 4,
+                    "macAddress": "34:5d:a8:0e:20:b4"
+                } ]
+        Example:
+            functions = Accesspoint(module)
+            final_input_data = functions.compare_ap_cofig_with_inputdata(all_apconfig)
+        """
+        apconfig = self.camel_to_snake_case(apconfig)
+        final_apchange = []
+        for each_input in self.payload["config"]["config_devices"]:
+            for eachap in apconfig:
+                if each_input.get("mac_address") == eachap["mac_address"] or \
+                    each_input.get("hostname") == eachap["ap_name"]:
+                    newdict = {}
+                    allkey = list(each_input.keys())
+                    for value in ("mac_address","hostname","management_ip_address", "family"):
+                        if value in allkey: allkey.remove(value)
+                    for each_key in allkey:
+                        if each_key == "ap_name_new":
+                            if each_input["ap_name_new"] != eachap.get("ap_name"):
+                                newdict["apNameNew"] = each_input["ap_name_new"]
+                        elif each_key == "ap_name":
+                            newdict[self.keymap[each_key]] = each_input[each_key]
+                        else:
+                            if each_input[each_key] != eachap[each_key]:
+                                newdict[self.keymap[each_key]] = each_input[each_key]
+                    if newdict.get("apName") is not None and newdict.get("apNameNew") is None:
+                        del newdict["apName"]
+                    if newdict:
+                        newdict["macAddress"] = eachap["eth_mac"]
+                        final_apchange.append(newdict)
+        if len(final_apchange) > 0:
+            return final_apchange
+        else:
```

Comment:
else is not needed at all if 'if' condition satisfy then it will return from there only. In next line you just log the message and return from there only.